Assets/Scripts/HealthBar.cs

```diff
@@ -24,14 +27,30 @@ private void Update()
         HealthBarFill();
         ChangeColor();
 
-        _healthText.text = Mathf.RoundToInt(_unitStats.Health).ToString();
+        if (_unitStats.Health <= 0)
+        {
+            _healthText.text = "DEAD";
+        }
+        else
+        {
+            _healthText.text = Mathf.RoundToInt(_unitStats.Health).ToString();
+        }
+
+        if (_unitStats.Health >= _unitStats.MaxHealth)
+        {
+            _healthBar.SetActive(false);
```

Comment:
You can simplify these lines: _healthBar.SetActive(_unitStats.Health < _unitStats.MaxHealth);

Reply:
Done